### PR TITLE
Footer: Set year for copyright automatically

### PR DESCRIPTION
**Pull request by fhemberger**
Little update to #464 and #465, so we don't have to update the footer each year. ;)

### CODE REVIEW

**Comment by Knighton910** [comment]
That makes sense, nice work @fhemberger dynamically :+1:

File: scripts/helpers/copyright-year.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function () {
+
+    return (new Date()).getFullYear();
```
Comment (stevemao) [inline scripts/helpers/copyright-year.js]: This should probably be `getUTCFullYear()`

Comment (fhemberger) [inline scripts/helpers/copyright-year.js]: @stevemao Oh, you're right. Thanks.

**Comment by phillipj** [comment]
Just pushed some style issues introduced with this PR in 87db4eea4d83597f8dfdc1f5037df6db1e5a94da, too bad we can't get Travis up-n-running again due to org restrictions :(

**Pull request merged**